cmake/fficonfig.h.in

```diff
@@ -28,14 +28,14 @@
 /* #undef FFI_NO_STRUCTS */
 
 /* Define to 1 if you have `alloca', as a function or macro. */
-#define HAVE_ALLOCA 1
+#cmakedefine HAVE_ALLOCA 1
```

Comment:
c42c3cd got us up to what autoconf does. de56297 alloca exists in stdlib.h in BSD derivs